cityofaustin/atd-vz-data
# Show a banner at the top of the screen when using a non-production environment

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/15892

## Testing
**URL to test:** 
Local and netlify. https://deploy-preview-1414--atd-vze-staging.netlify.app/#/crashes

**Steps to test:**
1. Check out the netlify link, youll see a banner at the top saying you are in a staging environment.
2. Check out the branch locally and start up your local DB and VZE, you will see a banner saying you are in a local environment.
3. Lmk thoughts on color preferences for the banner.


---
#### Ship list
